### PR TITLE
Updated 91_version_history file for 2.5.0 release

### PR DESCRIPTION
**Pull request by Mohit04tomar**

### CODE REVIEW

**Review by MarkRiddoch — APPROVED** [review]

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -33,100 +33,73 @@ Release Date: 2024-06-26
...
+       - A new tuning option has been added to the core that tunes the size of the cache maintained by the configuration manager. Increasing this cache size can speed up the startup of the system when large numbers of services are used within the system.
        - A new tuning option has been added to the north service to control the frequency of updating the stream position when writing data to the north. More details are available in the tuning section of the documentation.
-       - The documentation on securing FogLAMP has been updated.
+       - A new tuning option has been added to the north service that allows the number of buffers that are prefetched to be tuned.
+       - A small performance enhancement has been made in the south service such that the rate of storing statistics is not dependent on the ingest rate of the service. This will improve the performance of south services with high ingest rates.
+       - A performance enhancement has been added to all SQLite storage plugins that results in higher throughput and lower latency in all cases when using SQLIte as a storage engine.
```
Comment (RVerhoeff) [inline docs/91_version_history.rst:41]: Casing: SQLIte -> SQLite (second instance of SQLite in this sentence)

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -33,100 +33,73 @@ Release Date: 2024-06-26
...
 
 
     - Bug Fix:
 
+       - A typo in the statistics description for “Readings Sent North” has been fixed.
        - An issue that may occasionally cause the storage service to fail during purge when using the default SQLite storage engine for readings data storage has been resolved.
-       - The configuration category documentation has been updated to include more explanation and examples of the use of the various list type configuration items.
        - A problem with integer overflow in the sqlite storage engine has been resolved.
-       - Some changes have been done to the way the API handles passwords to make them more secure.
+       - An issue with one of the API entry points that could allow for command injection into the underlying operating system has been resolved.
```
Comment (RVerhoeff) [inline docs/91_version_history.rst:61]: In line 60: Casing: sqlite storage engine -> SQLite storage engine

**Review by RVerhoeff — CHANGES_REQUESTED** [review]
We should have consistent casing of _SQLite_

File: docs/91_version_history.rst [hunk 3]
```diff
@@ -33,100 +33,73 @@ Release Date: 2024-06-26
...
        - The system now tracks failed login attempts and can block accounts that have excessive failed attempts to login.
+       - A new security feature has been added to disconnect idle sessions after a configurable time.
        - Support for adding list names to configuration category items of type list has been added.
-       - Documentation has been added to the 'Tuning Fledge' section to describe the performance counters in the storage layer and also give some general tips on the subject of using the performance counters to tune the Fledge installation.
-       - 
-       - The storage performance monitors have been updated to include the table name in the monitor name. This allows tracking of which tables are being heavily used within the system.
-       - A new unable parameter has been added to the storage service to configure the number of threads that will be used to interact with the buffered reading data. This limits that impact on heavily loaded systems and also allows for the threads to be pre-created, which slightly reduces the latency.
+       - Support bundles and backups have been updated to require extra privileges to access
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:49]: end the sentence with full stop

**Review by MarkRiddoch — APPROVED** [review]

**Review by RVerhoeff — APPROVED** [review]

**Pull request merged**